src/app/faq/page.tsx

```diff
@@ -1,3 +1,4 @@
+"use client";
```

Comment:
To check - was there any reason why this isn't added to the other pages?

Reply:
All the pages have this (see src/app/faq/page.tsx, src/app/mentors/page.tsx, src/app/page.tsx)